starter-code/scripts/models/article.js

```diff
@@ -46,12 +46,19 @@ Article.fetchAll = function() {
     /* When our data is already in localStorage:
     1. We can process and load it,
     2. Then we can render the index page.  */
+    Article.loadAll(JSON.parse(localStorage.getItem('blogArticles')));
```

Comment:
Nice use of the localStorage method! What else could we have done to accomplish this?